### DISCUSSION TITLE
class Annotation

### DISCUSSION THREAD

**Discussion by MoraruMaxim** [LLNL/Caliper#519] [Q&A]
Hello,
I am trying to create a custom context attribute and use it to mark a region (`begin()/end()`). I think I am missing something, because when I try to obtain the runtime information on this region, I get an empty response. 

My program :
```
...
cali::Annotation phase_ann("test");
phase_ann.begin("Initialization");
some_function();
phase_ann.end();
...
```

My config:
```
CALI_SERVICES_ENABLE=aggregate,event,report,timestamp
CALI_REPORT_CONFIG="select test,count(),sum(sum#time.duration) as \"Exclusive time\"
```

The output I get:
```
count Exclusive time 
    1                
    1   
```

**Comment by daboehme** [comment]
Hi @MoraruMaxim ,

This should in principle work. Which Caliper version are you using?

For the time you'll need to specify `sum(sum#time.duration.ns)` (note the `.ns`) since Caliper v2.10. However I'm not sure why the `test` column is missing in the output. Can you check the `CALI_REPORT_CONFIG` line in your config again? You're missing a `"` at the end but even then it worked in my test. If you can, please send me a small reproducible example.

**Comment by MoraruMaxim** [comment]
Thank you for this quick response. I was using Caliper v2.10. Hence adding ".ns" solved my problem. The "test" column is printed. I think I just miss-copied the output.
